features/expectations/body_json_example.feature

```diff
@@ -74,6 +74,7 @@ Feature: Body - JSON example
     }
     """
     Then Gavel will NOT set any errors for "body"
+    And field "body" is valid
```

Comment:
Did we agree these can be squashed into one line?

```
Then Gavel will NOT set any errors for "body"
And field "body" is valid
```

or do you want to keep it separated everywhere like you have it now? I think I commented here on this topic earlier.

Reply:
It seems I've missed this line. Will update.

Reply:
Pushed in the forgotten commits.